src/hotspot/share/opto/generateOptoStub.cpp

```diff
@@ -261,7 +261,7 @@ void GraphKit::gen_stub(address C_function,
                                           frameptr(),
                                           returnadr());
   root()->add_req(_gvn.transform(to_exc));  // bind to root to keep live
-  C->init_start(start);
+  DEBUG_ONLY(C->verify_start(start);)
```

Comment:
This looks fine, but instead of marking every call site with DEBUG_ONLY, how about adding NOT_DEBUG_RETURN to the declaration of verify_start(), so it is a no-op in non-debug builds?  For an example, see check_no_dead_use().

Reply:
Thanks a lot for feedback @dean-long!